### ISSUE TITLE
Sprites swaps textures for a frame

### ISSUE THREAD

**Issue by holywyvern** [holywyvern/mv-pixi-upgrade#8]
There is some strange flickering with textures sometimes.

It happens on both map and battle.

More prominent with battlers.

**Comment by Hudell** [comment]
Consistently happens on a demo I have here.
My map displays 17 tiles on screen and I have an event on X = 17. When I move left and the map scrolls left, hiding that event, the spriteset flickers on screen.

**Comment by holywyvern** [comment]
It still happens sometimes, right?
So there may be a problem there for now...

**Comment by Hudell** [comment]
Yes, on webGL mode there are still other things flickering on the screen.
In canvas mode I didn't find anything else, but I haven't checked inside battles yet.

Canvas mode is currently running in stable 60fps, while webGL runs in 20~40fps for me.

**Comment by holywyvern** [comment]
Strange because after the tilemap plugin was made I am having an steady 60 fps.
We may need to considerer the specs of each pc in order to view if the tilemap made with shaders is fast. (of course, on intel chips without graphical acceleration it may be slower than a vanilla implementation)

**Comment by ivanpopelyshev** [comment]
Im a bit busy, but next time i'll be available, I'll fix EVERYTHING in this conversion.

**Comment by Hudell** [comment]
Oh, I didn't use the tilemap plugin yet.

**Comment by holywyvern** [comment]
Try to use it, it _will_ run better
